frontend/device/sdl/device.lua

```diff
@@ -59,6 +59,16 @@ function Device:init()
     Generic.init(self)
 end
 
+function Device:setTime(hour, min)
```

Comment:
I guess, I understand now. We need to add that function:
```

function Device:setTime(hour, min)
        return false
end
```
in: https://github.com/koreader/koreader/blob/master/frontend/device/generic/device.lua